src/components/Dataset/ShareDatasetButton.js

```diff
@@ -10,18 +11,24 @@ import { InlineMessage } from 'components/shared/InlineMessage'
 import { Modal } from 'components/shared/Modal'
 import { TextInput } from 'components/shared/TextInput'
 
-export const ShareDatasetButton = ({ datasetId }) => {
+export const ShareDatasetButton = ({ dataset }) => {
   const { openModal } = useModal()
   const { setResponsive } = useResponsive()
   const { startTimer, clearTimer } = useTimeoutInCallback(() => {
     setIsCopied(false)
   }, 3000)
   const [isCopied, setIsCopied] = useState(false)
   const [value, handdleCopy] = useCopyToClipboard(null)
```

Comment:
```suggestion
  const [value, copyText] = useCopyToClipboard(null)
```

and then update below